src/scripts/main.js

```diff
@@ -1 +1,93 @@
 'use strict';
+
+const firstPromise = () => {
```

Comment:
firstPromise = arrow function, which return Promise

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

it's shorter and clearer
const myPromise = new Promise((resolve, reject) => {
  ...
});